go.mod

```diff
@@ -31,6 +31,8 @@ require (
 	sigs.k8s.io/kubefed v0.1.0-rc2
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/MatousJobanek/toolchain-common v0.0.0-20191009091227-b631ea4cbdc3
```

Comment:
My regular reminder to myself so I don't forget to re-check it again when common change is merged.

Reply:
thanks for the reminder - done here: c50921c
* removed replace
* updated common
* ran `go mod tidy`